README.md

```diff
@@ -382,6 +383,29 @@ $ biko g s -q "How to configure biko"
 
 Blazing fast.
 
+### Google Workspace
+
+* Open Google Workspace search from your terminal.
+
+```
+$ biko googleworkspace [product] [flag(s)]
+# or
+$ biko gw [product] [flag(s)]
+```
+
+| Product      | What                       | Command                      | Flags(Optional) |
+| :----:       | :----:                     | :----:                       | :----:        |
+| drive        | Search on Google Drive     | `drive`, `dr`                | `--query, -q` |
+| document     | Search on Google Docs      | `document`, `dc`             | `--query, -q` |
+| document     | Create a new Google Docs   | `document`, `dc`             | `--new, -n`   |
```

Comment:
I would like to use `doc`  🙏 

```suggestion
| document     | Search on Google Docs      | `document`, `doc`             | `--query, -q` |
| document     | Create a new Google Docs   | `document`, `doc`             | `--new, -n`   |
```

Reply:
Thanks for your comment! fixed in e42cd96